package.yaml

```diff
@@ -144,6 +143,7 @@ executables:
     source-dirs: app
     dependencies:
       - fission-web-api
+      - selda-postgresql
```

Comment:
No longer a dependency for the app. This frees anything downstream (read: the Fission CLI) from the `lib-pg` dependency.